docs/source/kit/battery.rst

```diff
@@ -0,0 +1,76 @@
+is vital to treat such a high concentration of energy with respect. If
+you do not, there is a serious risk of fire and injury. To avoid this,
+you should follow the safety information on this page closely, at all
+times.
+
+.. Warning:: You must not use any batteries, chargers, bags or cables not explicitly authorised by SourceBots.
+
+Warnings
+--------
+
+-  Never leave batteries unattended when they are in use or charging.
+-  Always place the batteries in the provided charging bag when charging
+   or storing.
+-  Always follow the charging checklist precisely to make sure the
+   charger is correctly configured.
+-  Always leave the charging leads connected to the battery charger at
+   all times.
+-  Do not charge or use a damaged battery.
+-  If a battery has any cuts, nicks, exposed copper on wires or is
+   bulging to the point of no longer being squishy, contact us
+   immediately.
+
+.. Danger:: Disobeying the above warnings greatly increases the risk of damage to your battery, which can have **fatal consequences**. If you have any questions or doubts, talk to a member of staff immediately.
+
+Storing batteries
+-----------------
+
+When your batteries are not actively in use, they should be safely
+stored. You must disconnect the batteries from all electrical equipment,
+and place them in the battery charging bag. You should then store the
+charging bag in a safe location.
+
+Operating batteries
+-------------------
+
+To use your batteries, you must connect them to the power board. Do not
+tamper with the cable or connect the batteries to anything other than
+the power board (or the charger when charging).
+
+During operation, the battery is protected by over-current protection
+and a fuse in the power board. If any equipment is short circuited, the
+over-current protection will activate, protecting the battery. In
+extreme circumstances the fuse may blow to prevent damage to the
+battery. This is an important safety feature: do not, under any
+circumstances, bypass the fuse. It is not user serviceable and if it has
+blown then the power board must be replaced. If you suspect the fuse has
+blown then please contact us straight away.
+
+Mechanical damage to a battery can be dangerous, and a puncture or large
+force applied to a battery causes a serious risk of fire. To avoid this,
+your battery should be shielded from mechanical damage while you operate
+it. Secure your battery to your robot, so that it does not move or fall
+off while the robot moves. You should also build a compartment for the
+battery to be placed in, so that accidental collisions do not damage the
+battery.
+
+Flat batteries
+--------------
+
+When the battery has been almost completely discharged, the Power Board
+will automatically turn off and the LED marked “Power / Flat Battery
+Indicator” in the `diagram </kit/power-board>`__ will
+flash red and green. You should immediately disconnect the battery.
```

Comment:
Probably worth saying that the competitor should then ask a faciliator to exchange their flat battery for a fresh one, to avoid any ambiguity around whether they should be trying to charge it themselves or not.